_projects/12-documentation-improvements.md

```diff
@@ -4,8 +4,8 @@ name: Documentation Improvements
 web:
 github: https://github.com/scalacenter/exercises-scalatutorial
 origin: https://github.com/scalacenter/advisoryboard/blob/master/proposals/008-websites.md
-contributors: [julien, travis]
-status: Active
+contributors: [julien, jorge]
```

Comment:
Added @jvican thanks to his work on initiating effort to translate coursera course for Spanish, which I think is an awesome idea! Can revert this if you like Jorge. The status of the project is no longer active.